CONTRIBUTING.md

```diff
@@ -27,18 +27,16 @@ The [git-scm.com](https://git-scm.com/) site has links to download Git for your
 
 ### Where to get Node.js
 
+You can install Node.js + npm using a [Node.js version manager](https://docs.npmjs.com/cli/v7/configuring-npm/install#using-a-node-version-manager-to-install-nodejs-and-npm) (recommended).
+
 If you use Linux, use your operating system's package manager (`dnf`, `apt` or similar) to install the latest version of Node 14.
 If you use Ubuntu, you can also get a snap of the Node 14 program.
 
 If you use Windows or macOS, you can grab binaries from the [Node.js offical website](https://nodejs.org/).
```

Comment:
```suggestion
```

Let's drop these lines and leave only the Node version manager route in the docs.

Quote from the npm docs:

> **We strongly recommend using a Node version manager to install Node.js and npm.**